web/packages/teleterm/src/ui/DocumentTerminal/DocumentTerminal.tsx

```diff
@@ -31,7 +31,11 @@ import useDocTerminal, { Props } from './useDocumentTerminal';
 import { useTshFileTransferHandlers } from './useTshFileTransferHandlers';
 
 export default function DocumentTerminalContainer({ doc, visible }: Props) {
-  if (doc.kind === 'doc.terminal_tsh_node' && doc.status === 'disconnected') {
+  if (
+    doc.kind === 'doc.terminal_tsh_node' &&
+    'serverUri' in doc &&
+    doc.status === 'disconnected'
+  ) {
```

Comment:
`DocumentReconnect` is going to get completely rewritten in another commit in this PR. This change is just to satisfy the type system.